models/skilledImpactRequestsModel.js

```diff
@@ -30,5 +33,40 @@ async function findSkilledImpactRequests(campaign_id) {
         });
 }
 
-module.exports = { findSkilledImpactRequests };
+async function insertSkilledImpactRequests(skilledRequests, campaign_id) {
+    for (const skilledRequest of skilledRequests) {
+        const skillProps = ['skill', 'point_of_contact', 'welcome_message', 'our_contribution'];
```

Comment:
this doesn't need to be initialized inside the loop or even the function; it can go at the top of the file (see [this example](https://github.com/Lambda-School-Labs/key-conservation-be/pull/156/files#diff-bfb94d22a3518ce2e52c5ea76601284c) from David's PR)